app/bingey-ui/src/components/Watchlist/TitleCard/TitleDialog.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+import DialogUI from '@material-ui/core/Dialog';
+import Slide from '@material-ui/core/Slide';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import 'components/Watchlist/TitleCard/cardStyle.scss';
+
+const useStyles = makeStyles({
+  media: {
+    height: 260,
+  },
+});
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction='up' ref={ref} {...props} />;
+});
+
+export default function TitleDialog({ titles, onClose, open }) {
+  const handleCloseDialog = () => {
+    onClose();
+  };
+
+  const classes = useStyles();
+
+  return (
+    <div className='center centerFlex'>
+      <DialogUI
+        open={open}
+        onClose={handleCloseDialog}
+        TransitionComponent={Transition}
+        aria-labelledby='alert-dialog-slide-title'
+        aria-describedby='alert-dialog-slide-description'
+      >
+        <DialogTitle className='center big'>{titles.name}</DialogTitle>
+        <DialogContent>
+          <CardMedia
+            className={`center ${classes.media}`}
+            image={titles.image}
+            title='Movie poster'
```

Comment:
I don't think hardcoding _movie_ is a good decision. What if the title is a series